src/electron-forge-start.js

```diff
@@ -28,15 +28,19 @@ const main = async () => {
 
   dir = await resolveDir(dir);
   if (!dir) {
-    startSpinner.fail();
+    locateSpinner.fail();
     console.error('Failed to locate startable Electron application'.red);
     if (global._resolveError) global._resolveError();
     process.exit(1);
   }
+  locateSpinner.succeed();
```

Comment:
Perhaps this function should be split out so that the main function looks something like:

```javascript
async main () {
  // parse args
  await locate();
  await rebuild();
  await launch();
}
```